projects/microphysics/argo/end_to_end.py

```diff
@@ -248,12 +271,10 @@ def parameters(self):
         ml_config = deepcopy(self.ml_config)
         ml_config["out_url"] = model_out_url
 
-        prog_config = deepcopy(self.prog_config)
         model_path = os.path.join(model_out_url, "model.tf")
-        assert model_path.startswith("gs://")
-        prog_config["zhao_carr_emulation"] = {
-            "gscond": {"path": model_path, "gscond_cloud_conservative": True}
-        }
+        prog_config = set_prognostic_emulation_model(
+            self.prog_config, model_path, gscond_only=True, gscond_conservative=True
```

Comment:
The context isn't showing up in the diff. Just want to ensure this won't apply to gscond+precpd runs.

Reply:
fixed in a following commit.